pendulum_control/src/pendulum_demo.cpp

```diff
@@ -219,7 +219,8 @@ int main(int argc, char * argv[])
   options.memory_strategy = memory_strategy;
   // RttExecutor is a special single-threaded executor instrumented to calculate and record
   // real-time performance statistics.
-  auto executor = std::make_shared<pendulum_control::RttExecutor>(options);
+  rclcpp::Executor::SharedPtr singleThreadExecutor = std::make_shared<rclcpp::executors::SingleThreadedExecutor>();
```

Comment:
Let's keep the original functionality: instance of the `RttExecutor`

Reply:
As discussed, allow passing a `nullptr` to the constructor.

Reply:
Done